src/data/bucket/fill_extrusion_bucket.js

```diff
@@ -225,8 +212,8 @@ function isBoundaryEdge(p1, p2) {
 }
 
 function isEntirelyOutside(ring) {
-    return ring.every(p => p.x < 0) ||
-        ring.every(p => p.x > EXTENT) ||
-        ring.every(p => p.y < 0) ||
-        ring.every(p => p.y > EXTENT);
+    for (const p of ring) {
```

Comment:
This isn't testing the same thing. For example, a ring with xs all < 0 should return `true`, but this logic will return `false`, because x < 0 ⇒ x <= EXTENT.

Reply:
Good catch, fixed.